packages/library/src/commonMain/kotlin/io/realm/MutableRealm.kt

```diff
@@ -23,9 +23,17 @@ import kotlinx.coroutines.flow.Flow
 import kotlin.reflect.KClass
 
 /**
- * This class represents the writeable state of a Realm file. The only way to modify data in a Realm is through
- * instances of this class. These are provided and managed automatically through either [Realm.write] or
+ * Represents the writeable state of a Realm file.
+ *
+ * The only way to modify data in a Realm is through instances of this class.
```

Comment:
```suggestion
 * To modify data in a [Realm], use instances of this class.
```
Passive -> active rewrite.